ambuda/models/proofing.py

```diff
@@ -89,6 +89,16 @@ class Page(Base):
     #: for edit conflicts. See the `add_revision` function for details.
     version = Column(Integer, default=0, nullable=False)
 
+    #: A raw-ish version of the Google OCR response. We store the response as a
+    #: list of word-level bounding boxes in the following format:
+    #:
+    #:     x1 y1 x2 y2 text
+    #:
+    #: The field is nullable so that we can distinguish between (1) a page that
+    #: has no OCR data and (2) a page whose OCR results are empty, such as if
+    #: the page is blank.
+    ocr_bounding_boxes = Column(Text_, nullable=True)
```

Comment:
How do we make the association between this column and wherever the actual text is stored?

Reply:
Discussed with @shreevatsa offline and this scheme should be workable for now. Since it's also somewhat speculative, we can always just change it later.